aten/src/THCUNN/HardTanh.cu

```diff
@@ -10,9 +10,7 @@ struct hardtanhupdateOutput_functor
   const T min_val_;
 
   hardtanhupdateOutput_functor(T min_val, T max_val)
-    : min_val_(min_val)
```

Comment:
isn't this the same order? what exactly changed?

Reply:
was min, max, now max, min.

Reply:
ohh, i see. thanks. idk how I misread it